### PR TITLE
🧹 Uncomment `PG*` default lines in `.env.example`

### PR DESCRIPTION
**Pull request by daltonrpruitt**
In general, people seem to need these to be uncommented more often than not. 
Discussion seemed to indicated this was a desirable change.

### CODE REVIEW

**Review by anaulin — APPROVED** [review]
I wonder if this particular bit is a Linux vs Mac Docker difference. I use Docker on Mac and I keep these variables commented out. Might be worth adding a comment to that effect as part of this PR (if we believe that to be the case)?

I don't have a strong opinion on this, and I'm happy with whatever makes folks' lives easier, and this is definitely safe to merge!

**Comment by rosschapman** [comment]
@daltonrpruitt, as @anaulin suggested, I think a short comment to make developers aware that this default is not always necessary or desirable depending on your machine  would be useful. What about adding to `CONTRIBUTING.md` at https://github.com/zinc-collective/convene/blob/main/CONTRIBUTING.md?plain=1#L76.

**Review by rosschapman — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 # If you are using docker to manage your postgresql database,
 # you will want to uncomment these
-# PGHOST=localhost
-# PGUSER=postgres
-# PGPASSWORD=password
+PGHOST=localhost
+PGUSER=postgres
+PGPASSWORD=password
```
Comment (zspencer) [inline .env.example:10]: I think we only want to uncomment the first two; as `password` is not guaranteed to be their postgresql password.

**Comment by anaulin** [comment]
I personally vote against adding new separate documentation pages, such as `SETUP_FAQ.md`. In my opinion spreading the documentation across different pages makes it even harder to find whatever you're looking for. I'd rather have better-organized longer documents where I can Ctrl-F and also see an idea of the contents at a glance.

**Comment by zspencer** [comment]
That's fair; Never underestimate the power of 🍀+F!

**Comment by rosschapman** [comment]
Some might argue that's a better way to write code as well. 😅

**Comment by anaulin** [comment]
> Some might argue that's a preferred way to write code as well. 😅

🙀🙅🏼‍♀️🚫

**Comment by anaulin** [comment]
@daltonrpruitt are you still planning to work on this PR, or shall we close it for now?

**Comment by anaulin** [comment]
@daltonrpruitt I'm going to close this PR to keep our number of open PRs to a minimum. Please feel free to reopen if/when that makes sense.

**Pull request closed without merging**